snap/snapcraft.yaml

```diff
@@ -301,26 +318,9 @@ parts:
       - -go*
     override-build: |
       set -eux
-      . build-scripts/set-env-variables.sh
 
-      REMOVE_KUBE_SNAP_BINS=false
-      echo "Reached here"
-      # if "${KUBE_SNAP_BINS}" exist we have to use the binaries from there
-      # if "${KUBE_SNAP_BINS}" does not exist but it is set we will put the k8s binaries there
-      # if "${KUBE_SNAP_BINS}" does not exist and it is not set we do not need to keep the created binaries
-      if [ ! -e "${KUBE_SNAP_BINS}" ]; then
-        if [ -z "${KUBE_SNAP_BINS}" ]; then
-          # "${KUBE_SNAP_BINS}" not set, we will remove the binaries after we build them.
-          REMOVE_KUBE_SNAP_BINS=true
-          . build-scripts/set-env-binaries-location.sh
-        fi
-        echo "Downloading binaries"
-        . build-scripts/fetch-other-binaries.sh
-        echo "Building k8s binaries"
-        . build-scripts/build-k8s-binaries.sh
-      else
-        echo "Binaries provided in $KUBE_SNAP_BINS"
-      fi
+      . build-scripts/set-env-variables.sh
+      KUBE_SNAP_BINS="/root/parts/k8s-binaries/build/build/kube_bins/$KUBE_VERSION"
```

Comment:
It seems like some of this script should be moved over to the `k8s-binaries` part. Is there any reason not to move over the bits related to copying the K8s binaries into the snap (and maybe switching to an `organize` directive while we're at it)?

Reply:
This absolute path does not look right. There must be a way to express this relative to one of the varibales in https://snapcraft.io/docs/parts-environment-variables and/or move the binary placement (eg `cp $KUBE_SNAP_BINS/$KUBE_ARCH/kube-apiserver .`) in the new part.

Reply:
Good catch. I put that in to just make it work while I investigated moving these binaries into their own steps as well. I don't think we'll need it as soon as that can happen.

Reply:
This line won't be necessary as soon as https://github.com/ubuntu/microk8s/pull/1490 gets merged